packages/ng/forms/rich-text-input/plugins/clear-format/clear-format.command.ts

```diff
@@ -0,0 +1,40 @@
+import { $createParagraphNode, $createTextNode, $getSelection, $isRangeSelection, $isTextNode, COMMAND_PRIORITY_NORMAL, createCommand, LexicalEditor } from 'lexical';
+import { $setBlocksType } from '@lexical/selection';
+import { replaceTextNodeContentPartWith } from '../../utils';
+
+export const CLEAR_FORMAT = createCommand<void>('CLEAR_FORMAT');
+
+export function registerClearFormat(editor: LexicalEditor) {
+	return editor.registerCommand(
+		CLEAR_FORMAT,
+		() => {
+			editor.update(() => {
+				const selection = $getSelection();
+
+				if ($isRangeSelection(selection)) {
+					const anchor = selection.anchor;
+					const focus = selection.focus;
+					const nodes = selection.getNodes();
+
+					nodes.forEach((node) => {
```

Comment:
destruct for readability ?
```suggestion
					const { anchor, focus, getNodes } = selection.anchor;

					getNodes().forEach((node) => {
```

Reply:
Could not desctruct `getNodes` without runtime errors regarding `this` context. Vanilla JS FTW 😅